src/sonic-config-engine/minigraph.py

```diff
@@ -645,27 +649,63 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
 #
 ###############################################################################
 
-def filter_acl_mirror_table_bindings(acls, neighbors, port_channels):
+def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
+    filter_acls = {}
     """
-        Filters out inactive front-panel ports from the binding list for mirror
-        ACL tables. We define an "active" port as one that is a member of a
-        port channel or one that is connected to a neighboring device.
+        If the asic role is BackEnd no ACL Table (Ctrl/Data/Everflow) is binded.
+        This will be applicable in Multi-NPU Platforms.
     """
+    if sub_role == 'BackEnd':
```

Comment:
should we define a marco for BackEnd and use it? I assume BackEnd is used somewhere else in the file as well?

Reply:
@ Updated to use macro.

---
In reply to: [421177634](https://github.com/Azure/sonic-buildimage/pull/4491#discussion_r421177634) [](ancestors = 421177634)